packages/fast-components-styles-msft/src/text-field/index.ts

```diff
@@ -34,7 +34,7 @@ const styles: ComponentStyles<TextFieldClassNameContract, DesignSystem> = (
             boxSizing: "border-box",
             borderRadius: toPx(designSystem.cornerRadius),
             padding: "10px",
-            margin: `${density(designSystem.designUnit * 3)(designSystem)} 0`,
+            margin: `0`,
```

Comment:
nit: I know this works, but we don't actually need back-ticks here and would prefer to use quotes in these cases instead.